client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const buildYourOwnCta = isEnabled( 'signup/design-picker-build-your-own' ) ? (
```

Comment:
[nit] I think we can use `&&` to short-circuit this logic, since we don't really need the falsy condition.